### PR TITLE
new feature: Adds ellipsis if text overflows

### PR DESCRIPTION
**Pull request by 0xadri**
see feature request https://github.com/twbs/bootstrap/issues/19314

### CODE REVIEW

**Comment by adrien-be** [comment]
Just realized my PR was on Bootstrap 3 ... :/

File: less/utilities.less [hunk 1]
```diff
@@ -53,3 +53,15 @@
...
+
+// Adds ellipsis if text overflows
+// -------------------------
+
+.ellipsis-if-text-overflows {
+  display: inline-block;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  -o-text-overflow: ellipsis;
```
Comment (cvrebert) [inline less/utilities.less]: For future reference, we use [Autoprefixer](https://github.com/postcss/autoprefixer), so explicit prefixes such as this should normally not be included in our Less/Sass code.

**Comment by cvrebert** [comment]
Indeed, we're no longer accepting new features into Bootstrap 3; see https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md#pull-requests

We'd consider such a pull request for Bootstrap 4 (branch `v4-dev`) though. Feel free to open a new PR for this for Bootstrap 4.

**Comment by adrien-be** [comment]
Thanks for the feedback. I'll send PR to V4 instead then.

Should the doc on https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md#pull-requests be corrected though?

So instead of:

> 2. If you cloned a while ago, get the latest changes from upstream:
> 
> `git checkout master`
> `git pull upstream master`

Rather:

> `git checkout v4-dev`
> `git pull v4-dev v4-dev`

--

And also change the point 5 to:

`git pull [--rebase] v4-dev v4-dev`

**Pull request closed without merging**